### DISCUSSION TITLE
Looking to create Collections instead of Playlists

### DISCUSSION THREAD

**Discussion by dchapNL** [Taxel/PlexTraktSync#134] [Ideas]
Thank you all for a fantastic submission...

As stated, looking to create Collections instead of Playlists. Plex's playlist GUI is very limited compared to Collections, and I'd like for shared users to have access to the new collections ... anyone aware of a way to make this happen?

**Comment by glensc** [comment]
@dchapNL can you describe what is the current Playlist integration?

**Reply by dchapNL** [reply]
@glensc When I run the script, and it syncs with liked lists I have in Trakt, it creates a Playlist on my Plex server.  I'm hoping to create a collection on my Plex server in lieu of a Playlist

**Reply by dchapNL** [reply]
The issue with playlists, is you can only see a list of movies and select one to play, you cannot see movie info, or browse the playlist with thumbnails, its just exactly what it says, a playlist, and not a library/collection useful for browsing
https://prnt.sc/10pbtbc

**Reply by glensc** [reply]
I saw some weird code, so created this issue:
- https://github.com/Taxel/PlexTraktSync/issues/124

perhaps it should be switched to collection then! perhaps as opt-in.

**Comment by simonc56** [comment]
I think this feature can be added as opt-in.
Keep in mind that :
- Playlists can have items sorted by ranks, Collections can not.
- Playlists are in user profile, Collections are in library itself. I think only admins can create Collections, not invited or managed users.

I suggest to separate this feature from the issue #124.

**Comment by gitttt54** [comment]
Hello,

I came here to ask if there is any progress on this feature "opionally create a plex collection from trakt lists instead of a plain playlist"

The feature would be a great addition for plex users, as explained here https://github.com/Taxel/PlexTraktSync/discussions/134#discussioncomment-500232 by dchapNL. Finding a movie in a plex playlist is so cumbersome and userunfriendly.

Having all the Trakt lists in a collection will give any plex user ease to select only the movie he/she/it wants to watch.

If this feature needs some testing, I'm glad to help and report.

Thanks anyway for this useful tool you made available for us.

**Comment by Amateur-God** [comment]
I also like this idea and would be happy to help test it, Idk if I've done something wrong I've set true for the playlist part in plextraktsync config however it is only syncing the default Trakt watch list to my Plex server not the extra lists I've created for language specific shows like Korean, Japanese etc. 

It would be helpful if the collections version when done syncs user created lists to collections aswell as the default Trakt watch list or to include in the config an area to set which list names to import so instead of just true/ false, something like `` If true import {comma seperated list name}``

**Comment by NGDM** [comment]
For future reference, requested in this FR: https://github.com/Taxel/PlexTraktSync/issues/1667

**Comment by glensc** [comment]
Closing in favor of feature request:
- https://github.com/Taxel/PlexTraktSync/issues/1667
